docs/tutorial/part-eight/index.md

```diff
@@ -1,94 +1,93 @@
+इस आखरी भाग में, आप साइट को तैयार करने के लिए कुछ सामान्य चरणों से गुजरने जा रहे हैं, आपको एक शक्तिशाली साइट डायग्नोस्टिक टूल से इंट्रोड्यूस करवाया जायेगा जिसका नाम [Lighthouse](https://developers.google.com/web/tools/lighthouse/) है। रास्ते में हम आपको कुछ और प्लगिन्स के साथ परिचित करवाएंगे जिनका आप अपने Gatsby साइट में कई बार इस्तेमाल करना चाहेंगे।
 
-## Audit with Lighthouse
+## Lighthouse के साथ ऑडिट करना
 
-Quoting from the [Lighthouse website](https://developers.google.com/web/tools/lighthouse/):
+[Lighthouse वेबसाइट](https://developers.google.com/web/tools/lighthouse/) से कोट करते हुए:
 
-> Lighthouse is an open-source, automated tool for improving the quality of web pages. You can run it against any web page, public or requiring authentication. It has audits for performance, accessibility, progressive web apps (PWAs), and more.
+> Lighthouse एक ओपन-सोर्स, ऑटोमेटेड टूल है जो आपको वेब पेजेज की गुणवत्ता बढ़ाने मैं मदद करती है। आप इसे किसी भी वेब पेज, पब्लिक या रिक्वायर्ड ऑथेंटिकेशन पर चला सकते हैं। इसमें परफॉरमेंस, अक्सेसिब्लिटी, प्रोग्रेसिव वेब अप्प्सस (PWAs), आदि के लिए ऑडिट्स है।
 
-Lighthouse is included in Chrome DevTools. Running its audit -- and then addressing the errors it finds and implementing the improvements it suggests -- is a great way to prepare your site to go live. It helps give you confidence that your site is as fast and accessible as possible.
+Lighthouse, Chrome DevTools मे शामिल है। इसकी ऑडिट चलाना -- और फिर इसके द्वारा पहचाने हुए त्रुटियों को ठीक करना और इसके द्वारा सुझाये गए सुधार को लागू करना -- एक बहुत सटीक तरीका है आपके साइट को लाइव लेजाने की तैयारी करने में। यह आपको यह विश्वास दिलाने में मदद करता है कि आपकी साइट यथासंभव तेज और सुलभ है।
 
-Try it out!
+इसको आज़माइये!
 
-First, you need to create a production build of your Gatsby site. The Gatsby development server is optimized for making development fast; But the site that it generates, while closely resembling a production version of the site, isn't as optimized.
+सबसे पहले, आपको अपने Gatsby साइट के लिए एक प्रोडक्शन बिल्ड बनाना पड़ेगा। Gatsby डेवलपमेंट सर्वर को आपका डेवलपमेंट तेज़ बनाने के लिए अनुकूलित किया गया है।; लेकिन जो साइट वह उत्पन्न करता है, जबकि वह साइट का प्रोडक्शन वर्शन जैसा लगता है, परन्तु वह उतना अनुकूलित नहीं हैं।
 
-### ✋ Create a production build
+### ✋ एक प्रोडक्शन बिल्ड बनाइये
 
-1.  Stop the development server (if it's still running) and run the following command:
+1.  डेवलपमेंट सर्वर को रोक दीजिये (अगर वह अब भी चल रहा है) फिर यह कमांड्स चलाइये:
 
 ```shell
 gatsby build
 ```
 
-> 💡 As you learned in [part 1](/tutorial/part-one/), this does a production build of your site and outputs the built static files into the `public` directory.
+> 💡 जैसा की आपने [भाग १](/tutorial/part-one/) मे सीखा, यह आपके साइट का प्रोडक्शन बिल्ड करता है और `public` डायरेक्टरी के अंदर बिल्ड स्टैटिक फाइल्स की आउटपुट डालता है।
```

Comment:
`फाइल्स की आउटपुट डालता ` => `फाइल्स को आउटपुट करता `

Reply:
`आउटपुट करताा है।` => `आउटपुट करता है।`

you have added two "matras" in karta

Reply:
@arshadkazmi42 Hey! Changes done.